federation/cmd/federation-apiserver/app/server.go

```diff
@@ -119,8 +121,13 @@ func Run(s *genericoptions.ServerRunOptions) error {
 	if err != nil {
 		glog.Errorf("Failed to create clientset: %v", err)
 	}
-	admissionController := admission.NewFromPlugins(client, admissionControlPluginNames, s.AdmissionControlConfigFile)
+	sharedInformers := informers.NewSharedInformerFactory(client, 10*time.Minute)
```

Comment:
> You mean on ServerRunOptions?

Hrm, now I remember why we're unable to use these paths to build the server downstream.  We have come in a layer deeper to provide a completely separate admission chain.